metadata/src/main/java/edu/unc/lib/dl/xml/FOXMLJDOMUtil.java

```diff
@@ -138,7 +139,7 @@ public static Element getDatastream(final Document foxml, final String id) {
     }
 
     public static List<Element> getFileLocators(final Document foxml) {
-        List<Element> result = new ArrayList<Element>();
```

Comment:
even though this file is about to go away, you just couldn't let it go, could you ;-)